keyboards/boardrun/bizarre/keymaps/default/keymap.c

```diff
@@ -26,17 +26,17 @@ enum layer_names {
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
     /*Bizarre Base Layer (defaults for "any" layout)
```

Comment:
Why was this file changed?  Specifically, why were the box-drawing characters removed?

Reply:
Answered via discord